transmart-core-db/grails-app/domain/org/transmartproject/db/i2b2data/PatientDimension.groovy

```diff
@@ -35,6 +35,7 @@ class PatientDimension implements Patient {
     // private
     String sourcesystemCd
     String sexCd
+    String inTrialId
```

Comment:
Hm, makes me wonder if this would work with querying, if you could query with a field constraint on `inTrialId`. Maybe we should just expose `sourceSystemCd` instead...